src/test/resources/schemas/inCollection.json

```diff
@@ -21,6 +21,10 @@
       "label": {
         "type": "string"
       }
-    }
+    },
+    "required": [
+      "id",
+      "label"
```

Comment:
Didn't we discuss this recently. I think, we should either have `"type": "Collection"` as const for all `inCollection` statements or remove the type statement everywhere for sake of consistency.